jwst/resample/resample.py

```diff
@@ -342,9 +356,7 @@ def resample_many_to_many(self, input_models):
                 log.info(f"Saved model in {output_name}")
                 output_models.append(output_name)
             else:
-                output_models.append(output_model.copy())
-            output_model.data *= 0.
-            output_model.wht *= 0.
+                output_models.append(output_model.data)
```

Comment:
```suggestion
                output_models.append(output_model)
```
If I run `strun jwst.resample.ResampleStep some_asn --single=True` I'm seeing a failure:
```
AttributeError: 'numpy.ndarray' object has no attribute 'meta'
```
I think it's because `output_models` here is a list of arrays instead of models. It also looks like we don't have a regtest for this.
If I apply the above suggestion the command succeeds. Does this interfere with other parts of this PR?

Reply:
I second this suggestion!  It would be useful for the future if we also need to pass around errors (#8828).

Reply:
This change was left over from a previous draft of this PR where I thought we would change the behavior of `resample_many_to_many`.  It turns out that for outlier detection we are now bypassing this function call, so it shouldn't hurt to revert the behavior.  Will change and test

Reply:
should be fixed in the new version